crates/rspack/tests/tree-shaking/webpack-inner-graph-export-default-named/snapshot/new_treeshaking.snap

```diff
@@ -158,7 +158,7 @@ __webpack_require__.r(__webpack_exports__);
 
 it("should generate valid code", ()=>{
     expect((0, _a__WEBPACK_IMPORTED_MODULE_0__["default"])()).toBe("x");
-    expect(new (0, _d__WEBPACK_IMPORTED_MODULE_3__["default"])().method()).toBe("x");
+    expect(new _d__WEBPACK_IMPORTED_MODULE_3__["default"]().method()).toBe("x");
```

Comment:
I think that the PR wouldn't affect the output, yet it's actually happening...

However, it seems to be acceptable.